RecoTracker/LSTCore/src/alpaka/LST.cc

```diff
@@ -236,8 +236,8 @@ void LST::getOutput(LSTEvent& event) {
   out_tc_seedIdx_.clear();
   out_tc_trackCandidateType_.clear();
 
-  auto const hits = event.getHitsInCMSSW<HitsSoA>(false);  // sync on next line
-  auto const& trackCandidates = event.getTrackCandidatesInCMSSW();
+  auto const hits = event.getHits<HitsSoA>(/*inCMSSW*/true, /*sync*/false);  // sync on next line
+  auto const& trackCandidates = event.getTrackCandidates(/*inCMSSW*/true, /*sync*/true);
```

Comment:
This is unrelated to your change, but I just noticed that this is a reference to a temporary, so Andrea might complain about it. I'll check where else we have this.

Reply:
I will fix also this.

Reply:
Seems like we're only doing this for track candidates. We should change the function signature to return just the view instead of a reference to the view.

Reply:
the method is declared as `const TrackCandidatesConst& LSTEvent::getTrackCandidates`
It returns `trackCandidatesHC_.value().const_view()`, which is implemented as `  ConstView const& const_view() const { return view_; }`

In all other cases where we return a copy we just use the fact that the copies are relatively light (or/and that we didn't plug that in a very tight loop)

Reply:
Oh I see, I didn't realize it was returning a reference.